packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Primitives/ScmKnownParameters.cs

```diff
@@ -28,6 +29,15 @@ internal static class ScmKnownParameters
         public static readonly ParameterProvider OptionalOptions = new("options", FormattableStringHelpers.Empty, nullableModelReaderWriterOptionsType, DefaultOf(nullableModelReaderWriterOptionsType));
         public static readonly ParameterProvider JsonElement = new("element", FormattableStringHelpers.Empty, typeof(JsonElement));
         public static readonly ParameterProvider Data = new("data", FormattableStringHelpers.Empty, typeof(BinaryData));
+        public static readonly ParameterProvider ApiKeyAuth = new ParameterProvider(
+            "credential",
+            $"A credential used to authenticate to the service.",
```

Comment:
```suggestion
            "A credential used to authenticate to the service.",
```

Reply:
The description parameter is of type `FormattableString`.